### PR TITLE
wdio env var test

### PR DESCRIPTION
**Pull request by kb-kerem**
> Issue : IRIS-1051

## Description
Adds env var integration tests to wdio workflow

## Types of Changes

- New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: visual-js/visual-wdio/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,28 @@
+FROM node:18 AS runner
```
Comment (FriggaHel) [inline visual-js/visual-wdio/Dockerfile:1]: Shouldn't we mainly test on the latest LTS ?

**Review by FriggaHel — COMMENTED** [review]
Are the `integration-tests` folder automatically excluded from npm package ?

**Comment by kb-kerem** [comment]
> Are the `integration-tests` folder automatically excluded from npm package ?
Nope, [this](https://github.com/saucelabs/visual-sdks/pull/75/files#diff-50cda7a317524808331d3dc29885e894603d62f7a5b28e08905540836b4756dcL5) was done to exclude

File: visual-js/visual-wdio/Dockerfile (hunk 1 above)
Comment (kb-kerem) [reply visual-js/visual-wdio/Dockerfile:1]: we use node 18 on build pipeline, so it's probably fair to stick to the same here

**Review by FriggaHel — APPROVED** [review]

**Pull request merged**